libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/lifecycle/UICoordinator.kt

```diff
@@ -37,14 +37,14 @@ abstract class UICoordinator<T : ViewGroup>(
                 attachedObserver = viewBinder.bind(viewGroup)
                 attachedObserver?.onAttached(mapboxNavigation)
             }
-        }.invokeOnCompletion {
-            attachedObserver?.onDetached(mapboxNavigation)
```

Comment:
This causes the `MapboxNavigation` instance to become `null` before `onDetached` is called